src/SFA.DAS.EmployerIncentives.Data.UnitTests/SFA.DAS.EmployerIncentives.Data.UnitTests.csproj

```diff
@@ -11,11 +11,11 @@
     <PackageReference Include="FluentAssertions" Version="5.10.3" />
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.5" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="3.1.5" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
     <PackageReference Include="Microsoft.SqlServer.DACFx" Version="150.4769.1" />
     <PackageReference Include="Moq" Version="4.14.2" />
     <PackageReference Include="NUnit" Version="3.12.0" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.16.1">
+    <PackageReference Include="NUnit3TestAdapter" Version="3.15.1">
```

Comment:
Should the version of this be regressed? Does 3.16.1 introduce an issue

Reply:
Not sure why this was done, KS appeared to have done it as part of EI-442 consolidate packages. Probably aligning packages across the solution